File: 0283 move-zeroes/solution.rs
impl Solution {
    pub fn move_zeroes(nums: &mut Vec<i32>) {
        let mut j=0;
        for i in 0..nums.len() {
            if nums[i]!=0 {
                nums.swap(i,j);
                j+=1;
            }
        }
    }
}

File: 0283 move-zeroes/solution.cs
public class Solution {
    public void MoveZeroes(int[] nums) {
        int l=0;
        for(int i=0;i<nums.Length;i++) {
            if(nums[i]!=0) {
                int temp = nums[l];
                nums[l++] = nums[i];
                nums[i] = temp;
            }
        }
    }
}
